sdk/cpp/core/src/codec_service.cpp

```diff
@@ -66,22 +64,15 @@ CodecService::encode(CodecServiceProvider & provider, Entity & entity, bool pret
         XmlSubtreeCodec codec{};
         return codec.encode(entity, root_schema);
     }
-    try
```

Comment:
Is there a reason for eliminating the try catch around this code?

Reply:
This hid the real issue from the user. User can have try-catch in their app